static/app/components/events/eventReplay/replayContent.tsx

```diff
@@ -49,6 +49,18 @@ function ReplayContent({orgSlug, replaySlug, event}: Props) {
     return 0;
   }, [eventTimestamp, startTimestampMs]);
 
+  if (fetchError) {
+    return (
+      <Alert type="info" showIcon>
+        {t('The replay associated with this event could not be found.')}
+      </Alert>
+    );
+  }
+
+  if (onboardingPanel.enabled && !replayRecord) {
+    return <ReplayOnboardingPanel {...onboardingPanel} />;
+  }
```

Comment:
I believe we can never have a `replayRecord` when `onboardingPanel.enabled` is true, so the 2nd part of the condition could be extra.

```suggestion
  if (onboardingPanel.enabled) {
    return <ReplayOnboardingPanel {...onboardingPanel} />;
  }
```

and we could probably put this `return` earlier than the error case. The `useReplayData` call happens unconditionally even if `replayId` is the empty string.